File: shared/enums.py
from enum import StrEnum


class Gender(StrEnum):
    MALE = "male"
    FEMALE = "female"


class Language(StrEnum):
    RU = "ru"
    EN = "en"


class NsfwLevel(StrEnum):
    SAFE = "safe"
    ADULT = "adult"

File: internal/shared/enums.go
package shared

type Gender string

const (
	GenderMale   Gender = "male"
	GenderFemale Gender = "female"
)

type Language string

const (
	LanguageRU Language = "ru"
	LanguageEN Language = "en"
)

type NsfwLevel string

const (
	NsfwLevelSafe  NsfwLevel = "safe"
	NsfwLevelAdult NsfwLevel = "adult"
)
